Modules/zipimport.c

```diff
@@ -379,21 +405,29 @@ find_loader(ZipImporter *self, PyObject *fullname, PyObject **namespace_portion)
     return FL_MODULE_FOUND;
 }
 
+/*[clinic input]
+zipimport.zipimporter.find_module
+
+    fullname: unicode
```

Comment:
Yeah, we prefer to not change semantics while doing an Argument Clinic conversion (we have run into issues in the past when doing both at the same time).